src/components/ComponentDetails/EventTable/EventDescription/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import "./index.css";
+
+const EventDescription: React.FC<{
+  description: string;
+  deprecation: string | undefined;
+}> = ({ description, deprecation }) => (
+  <div className="event-description">
+    {description && <ic-typography>{description}</ic-typography>}
```

Comment:
Does description return an empty string if deprecated is set? If so, I think we should use `!!` so its truthy or falsy.